### PR TITLE
Add bash helper function to get pod logs

### PR DESCRIPTION
**Pull request by shashwathi**
Hope this helps with debugging test failures. 

Credit to @markusthoemmes work on knative/serving [PR]( https://github.com/knative/serving/pull/2432)

### CODE REVIEW

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -29,6 +29,25 @@
 source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
 source $(dirname $0)/e2e-common.sh
 
+# Helper functions.
+function dump_app_logs() {
```
Comment (adrcunha) [inline test/e2e-tests.sh]: This is nice. We should move this to the helper script so every repo can use it.

Comment (shashwathi) [reply test/e2e-tests.sh]: I agree completely

Comment (adrcunha) [reply test/e2e-tests.sh]: Filed https://github.com/knative/test-infra/issues/252 so we don't forget. :)

**Pull request merged**